### PR TITLE
Sort flags and add covid_vaccine_scheduling_frontend

### PR DESCRIPTION
**Pull request by mpelzsherman**
## Description of change
Adds a feature flag that will be used to toggle the availability of covid-19 vaccine scheduling links in the facility locator frontend. Also sorts the feature flags using http://yaml-sorter.herokuapp.com/.

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/22139

### CODE REVIEW

**Comment by mpelzsherman** [comment]
I’m willing to go through and change the descriptions back to using `>` if desired.

OTOH it’s nice to be able to run the sort again as needed.

I can also make the sorting a separate PR if folks insist.

**Review by thilton-oddball — CHANGES_REQUESTED** [review]
The addition of the feature flag looks good but I'd request that you remove all of the quotes and line feeds that were added. Though there are no hard and fast rules, quotes seem to generally not be used unless needed in YAML files. The quotes and line feeds that were added don't seem to be necessary and introduce inconsistency to the file as they were  applied to some but not all entries.

**Comment by mpelzsherman** [comment]
The quotes have been removed.

File: config/features.yml [hunk 1]
```diff
@@ -8,449 +8,374 @@
...
+  evss_upload_limit_150mb:
     actor_type: user
-    description: force the legacy urgent care path to use the new POS locator
-  facility_locator_ppms_forced_unique_id:
+    description: Allow user to upload files up to 150 MB in size (EVSS endpoint for 526 & CST)
+    enable_in_development: true
+  facilities_ppms_suppress_community_care:
     actor_type: user
-    description: Use an hexdigest for the ID on PPMS Place of Service Calls
+    description: Hide ppms community care searches"
```
Comment (thilton-oddball) [inline config/features.yml:111]: Trailing quote mark missed

File: config/features.yml [hunk 2]
```diff
@@ -8,449 +8,374 @@
...
   facilities_ppms_suppress_pharmacies:
     actor_type: user
-    description: Front End Flag to suppress the ability to search for pharmacies
-  facility_locator_predictive_location_search:
+    description: Front End Flag to suppress the ability to search for pharmacies"
```
Comment (thilton-oddball) [inline config/features.yml:114]: Trailing quote mark missed

File: config/features.yml [hunk 3]
```diff
@@ -8,449 +8,374 @@
...
     actor_type: user
-    description: Use predictive location search in the Facility Locator UI
-  facilities_ppms_suppress_community_care:
+    description: enable covid search/display"
+    enable_in_development: true
+  facility_locator_ppms_forced_unique_id:
     actor_type: user
-    description: Hide ppms community care searches
-  profile_schema_forms:
+    description: Use an hexdigest for the ID on PPMS Place of Service Calls"
```
Comment (thilton-oddball) [inline config/features.yml:121]: Trailing quote mark missed

File: config/features.yml [hunk 4]
```diff
@@ -8,449 +8,374 @@
...
+  facility_locator_ppms_legacy_urgent_care_to_pos_locator:
     actor_type: user
-    enable_in_development: true
-    description: >
-      Enables SchemaForm-based contact info edit forms on the VA.gov Veteran profile page
-  profile_show_receive_text_notifications:
+    description: force the legacy urgent care path to use the new POS locator"
```
Comment (thilton-oddball) [inline config/features.yml:124]: Trailing quote mark missed

File: config/features.yml [hunk 5]
```diff
@@ -8,449 +8,374 @@
...
+  facility_locator_predictive_location_search:
     actor_type: user
-    enable_in_development: true
-    description: >
-      https://www.va.gov/profile/ show Receive Text Notifications
-  va_online_scheduling:
+    description: Use predictive location search in the Facility Locator UI"
```
Comment (thilton-oddball) [inline config/features.yml:127]: Trailing quote mark missed

File: config/features.yml [hunk 6]
```diff
@@ -8,449 +8,374 @@
...
+  facility_locator_pull_operating_status_from_lighthouse:
     actor_type: user
+    description: A fast and dirty way to get the operating status from lighthouse"
```
Comment (thilton-oddball) [inline config/features.yml:130]: Trailing quote mark missed

File: config/features.yml [hunk 7]
```diff
@@ -8,449 +8,374 @@
...
-    actor_type: user
+      On https://www.va.gov/find-locations/ enable veterans to search for Community care by showing that option
+      in the "Search for" box.
     enable_in_development: true
-    description: >
-      Allows veterans to submit requests for VA appointments
-  va_online_scheduling_community_care:
+  facility_locator_show_operational_hours_special_instructions:
     actor_type: user
+    description: Display new field operationalHoursSpecialInstructions for VA facilities"
```
Comment (thilton-oddball) [inline config/features.yml:140]: Trailing quote mark missed

**Review by thilton-oddball — CHANGES_REQUESTED** [review]
There are a small handful of entries where a trailing quote mark was missed. If these can be cleaned up then I think we're good.

**Comment by mpelzsherman** [comment]
Sorry about those trailing quotes. I think I've got them all.

Those were introduced by the sorting tool I used. Hopefully folks will keep the keys sorted going forward so we won't have this issue again.

**Review by thilton-oddball — APPROVED** [review]
This all looks good now!

**Pull request merged**